cpp/include/cudf/io/orc.hpp

```diff
@@ -0,0 +1,832 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @file orc.hpp
+ * @brief cuDF-IO freeform API
+ */
+
+#pragma once
+
+#include "types.hpp"
```

Comment:
Looks like `types.hpp` is in the include dir.

```suggestion
#include <cudf/io/types.hpp>
```